release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -30,9 +30,364 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### Support for the spread operator in the list constructor
```

Comment:
```suggestion
##### Support for the `spread` operator in the` list` constructor
```

Reply:
@praneesha should we use backticks for these words? these are not ballerina keywords

Reply:
Better to use as it might be confusing with the verb `spread`.

Reply:
"List constructor expression" is a well-known construct in ballerina. For me, It is weird to highlight just the `list` part of it.
Same with spread operator. 
@MaryamZi thoughts?